File: backend/src/main/java/com/demo/student_management/dto/admin/PhanCongRequest.java
package com.demo.student_management.dto.admin;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PhanCongRequest {
    private String idGiaoVien;
    private String idLop;
    private String idMonHoc;
    private String idHocKy;
}

File: frontend/StudentManagement.Shared/Dtos/Admin/PhanCongRequest.cs
namespace StudentManagement.Shared.Dtos.Admin;

public sealed class PhanCongRequest
{
    public string IdGiaoVien { get; set; } = string.Empty;
    public string IdLop { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
}
